wurst/_handles/Unit.wurst

```diff
@@ -277,6 +277,12 @@ public function unit.removeAbility(int abil) returns boolean
 public function unit.removeItem(item itm)
 	UnitRemoveItem(this, itm)
 
+public function unit.removeItemById(int itemId)
+	for slot = 0 to this.inventorySize()-1
+		if this.itemInSlot(slot).getTypeId() == itemId
+			UnitRemoveItem(this, this.itemInSlot(slot))
```

Comment:
use extension func here.
Also add hotdoc to explain what exactly it does. (e.g. removing first occurance of the item in unit's inventory).
Maybe even return a boolean if an item has been removed.